hbar.go

```diff
@@ -0,0 +1,24 @@
+package hedera
+
+type Hbar struct {
+	tinybar int64
+}
+
+const max = int64(^uint(0) >> 1)
+const min = -max - 1
+
+var HbarMAX = Hbar{max}
```

Comment:
This should be `MaxHbar` according to Go convention, `ZeroHbar`, `MinHbar`, etc. 

See: https://golang.org/pkg/math/#pkg-constants